### PR TITLE
Allow extern interfaces https://github.com/TiVo/intellij-haxe/issues/202

### PR DESCRIPTION
**Pull request by as3boyan**
Allow extern interfaces
https://github.com/TiVo/intellij-haxe/issues/202

### CODE REVIEW

File: grammar/haxe.bnf [hunk 1]
```diff
@@ -258,7 +258,7 @@ externOrPrivate ::= 'extern' | privateKeyWord
 typedefDeclaration ::= macroClassList? externOrPrivate? 'typedef' componentName genericParam? '=' functionTypeWrapper ';'?
 {pin=5 mixin="com.intellij.plugins.haxe.lang.psi.impl.AbstractHaxeTypeDefImpl" implements="com.intellij.plugins.haxe.lang.psi.HaxeClass"}
 
-externClassDeclaration ::= macroClassList? externOrPrivate* 'class' componentName genericParam? inheritList? '{' externClassDeclarationBody '}'
+externClassDeclaration ::= macroClassList? externOrPrivate* ('class' | 'interface') componentName genericParam? inheritList? '{' externClassDeclarationBody '}'
```
Comment (as3boyan) [inline grammar/haxe.bnf]: Main change

**Comment by EBatTiVo** [comment]
You know, updating the copyright as a secondary change does not help when trying to decipher the differences in a change list.  Can you please combine both of these changes into a single change when you pull it?  Thanks.

Functionality approved for pull...  though I wonder if we should be making a new rule, rather than expanding the current one.  I'll leave it up to you.

**Comment by as3boyan** [comment]
I think https://github.com/TiVo/intellij-haxe/pull/204/files it's pretty clear

**Comment by EBatTiVo** [comment]
That is clear.  If github is going to make the pull look like that, then that makes me happy -- I'll have less work to do.  I guess we'll see how it actually shows up in the local repo once you make the pull.

**Pull request merged**